ui/src/components/AutoComplete2/Inputs/SingleInput/SingleInput.js

```diff
@@ -64,13 +65,14 @@ const SingleInput = ({
   return (
     <div className={wrapperClasses}>
       { shouldRenderSearch &&
-        <input
-          type="text"
+        <DebounceInput
+          debounceTimeout={377}
```

Comment:
@ht2 Many of DebounceInput in LL uses 377msec debounceTimeout. Should we give `700`?

Reply:
@h-kanazawa I think Andrew chose 377 back in the day because it was a fibonacci number (not really sure why that makes a difference). Any reason you think 700 might be better?

Reply:
@ryansmith94 "700" is not my thought. It is suggested by @ht2 
https://github.com/LearningLocker/enterprise/issues/969

> Debounce key entry (700ms)

Is 377 msec shorter than @ht2 expects?



(fibonacci ??)

Reply:
Hahaha, lets keep with 377 - it's quirky, easy to search for and actually feels quite a natural amount of time between key strokes (for me anyway....)